build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -1468,17 +1472,31 @@ my %hConfigDefine =
         },
     },
 
-    &CFGOPT_REPO_HOST =>
+    &CFGOPT_REPO_HARDLINK =>
```

Comment:
Why is CFGOPT_REPO_HARDLINK now repeated twice in this Data.pm (lines 1458 & 1475)?

Reply:
Yeah, looks like copied to be modified and then forgotten.